Jenkinsfile

```diff
@@ -1,29 +1,39 @@
 elifePipeline {
     def commit
+    stage 'Checkout', {
```

Comment:
Everything inside `elifeNode()` is performed on a separate node with Docker installed and a large disk to store images. We do `checkout scm` (which is a `git clone`) on both the main node and this separate `elife-libraries--*` one because both need access to either the repository files or Git metadata such as the commit sha.

Reply:
Hence should solve the failure of https://alfred.elifesciences.org/job/test-annotations/32